### DISCUSSION TITLE
Consider Switching To https://github.com/zalando/postgres-operator

### DISCUSSION THREAD

**Discussion by mitchross** [vehagn/homelab#107] [Ideas]
Consider swapping out your Postgres operator with 

https://github.com/zalando/postgres-operator/blob/master/docs/operator-ui.md
https://github.com/zalando/postgres-operator

You get a GUI to create and manage Postgres databases. Easier backups. Easier Monitoring. 

I am about 75% done https://github.com/mitchross/k3s-argocd-proxmox/tree/main/infra/database/cnpg-zalando

**Comment by vehagn** [comment]
Thank you for the suggestion!

I actually ran the Zalando Postgres-operator in [an earlier iteration of my homelab](https://github.com/vehagn/homelab/blob/kudeadm/infra/database/kustomization.yaml) after learning that the Norwegian Police use it. I wanted to try out CNPG after watching [this Kubecon Paris 2024 talk](https://youtu.be/l33pcnQ4cUQ) which tested and compared different database operators where CNPG came up on top.

Maybe I'll have to evaluate the Zalando one again.

**Reply by jfmatth** [reply]
I use Zalando as well and it's so so simple.  The documentation is much better than Crunchylabs, which is also a good product.
